worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -371,17 +371,29 @@ function isEmptyMap(mapValue) {
     return jsonString === '{}';
 }
 
+function isFieldValueEqualToValue(fieldValue, value) {
+    if (fieldValue.isStringValue() && fieldValue.getStringValue() === value) {
+        return true;
+    }
+    return false;
+}
+
 function fieldHasStringValue(document, fieldName, value) {
+    var fieldValues = document.getField(fieldName).getValues();
+    for(fieldValue of fieldValues) {
```

Comment:
Previously this line was `for (var fieldValue of fieldValues) {`.  Is there any reason the `var` has been dropped?  It would probably be better to be using `const` instead of `var` anyway for these loops, since you don't modify the value within the loops.